src/main/java/de/tum/in/www1/artemis/service/connectors/BambooService.java

```diff
@@ -1067,7 +1077,7 @@ private ResponseEntity retrieveArtifactPage(String url) throws BambooException {
      * @return true if the build plan id is valid.
      */
     @Override
-    public Boolean buildPlanIdIsValid(String buildPlanId) {
+    public Boolean buildPlanIdIsValid(String projectKey, String buildPlanId) {
```

Comment:
Can't we use `boolean` primitive type here? As I checked we only return true or false here

Reply:
Yes, I changed the header